operator/pkg/central/values/translation/translation.go

```diff
@@ -249,6 +251,30 @@ func getTelemetryValues(t *platform.Telemetry) *translation.ValuesBuilder {
 	return &tv
 }
 
+func getDeclarativeConfigurationValues(c *platform.DeclarativeConfiguration) *translation.ValuesBuilder {
+	declarativeConfig := translation.NewValuesBuilder()
+	if c == nil {
+		return &declarativeConfig
```

Comment:
shouldn't we put empty 
```
mounts:
      configMaps: []
      secrets: []
```
even in that case?

Reply:
Yeah I thought about this one as well, but looking at the other places we do something similar, it seems that's how we do things (e.g. telemetry values, secured cluster values). So instead of specifying any default values, we simply set it to nil and assume that the helm chart can handle if these values aren't set (which, it certainly can _and_ should).

Reply:
Sounds good. Do you think it makes sense to test this case in `translation_test`? If that's not too much work

Reply:
Ain't this what we test with[ the empty spec test](https://github.com/stackrox/stackrox/blob/65820256abbcb6cc2809f7a6ece8ca7729dd904a/operator/pkg/central/values/translation/translation_test.go#L52-L68) in there?
The test that covers that the helm chart can handle these "empty" values is within [the Helm chart PR](https://github.com/stackrox/stackrox/pull/4321/files#diff-453f50cb4de0a2077bd7029131c35907ed20743c69b421e26ed095396477e830R1-R6).